protoc-gen-swagger/options/openapiv3.proto

```diff
@@ -0,0 +1,207 @@
+syntax = "proto3";
+
+package grpc.gateway.protoc_gen_openapi.options;
+
+option go_package = "github.com/grpc-ecosystem/grpc-gateway/protoc-gen-swagger/options";
+
+import "google/protobuf/struct.proto";
+
+message OpenAPI {
```

Comment:
I would really like to see many more comments. OpenAPI itself can be a good source of comments.